web/onos-gui/src/app/onos-config/model.service.ts

```diff
@@ -14,38 +14,81 @@
  * limitations under the License.
  */
 
-import {Injectable} from '@angular/core';
-import {OnosConfigAdminService} from './proto/onos-config-admin.service';
+import { Injectable } from '@angular/core';
+import { OnosConfigAdminService } from './proto/onos-config-admin.service';
 import {
     ModelInfo,
 } from './proto/github.com/onosproject/onos-config/api/admin/admin_pb';
-import {Observable, Subscription} from 'rxjs';
-import {ErrorCallback} from './device.service';
+import { Observable, Subscription } from 'rxjs';
+import { ErrorCallback } from './device.service';
+
 
 @Injectable()
 export class ModelService {
     modelInfoList: ModelInfo[] = [];
     modelInfoSub: Subscription;
+    sortParams = {
+        firstColName: '',
+        firstCriteria: ModelService.modelSorterForward,
+        firstCriteriaDir: 0,
+        sortDirMap: new Map([
```

Comment:
I don't think this map is even needed now - all you have to do is track one column - all others should have no icon displayed

Reply:
removed